src/OrbitGl/MemoryTrack.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_MEMORY_TRACK_H_
+#define ORBIT_GL_MEMORY_TRACK_H_
+
+#include <string>
+
+#include "GraphTrack.h"
+#include "Track.h"
+
+class MemoryTrack final : public GraphTrack {
```

Comment:
Since your are doing this new class (which is awesome). Can you put this in namespace `orbit_gl`? It is currently somewhat annoying, because a few files in OrbitGl have the namespace and lots of them don't, but I think its the right direction for the future.

Reply:
Make sense! Changed accordingly. Thanks!